core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java

```diff
@@ -168,6 +175,23 @@
         return composite;
     }
 
+    /**
+     * Returns a system property composed of environment variables and system properties.
+     *
+     * <p>Properties present in system properties will
+     * take precedence over those in environment variables,
+     * similarly to spring boot behaviour.
```

Comment:
Please switch to `behavior`. Also remove the initial `<p>` tag, if possible.

Reply:
See previous comment (I didn't understood what exactly you mean by behaviour - so I hope it's ok ;) )